opennms-doc/guide-install/src/asciidoc/text/minion-custom-messaging-system/setup-kafka/steps-horizon.adoc

```diff
@@ -0,0 +1,37 @@
+
+==== Configure {opennms-product-name}
+
+// No section numbers for step-by-step guide
+:!sectnums:
+
+===== Step 1: Set Kafka as RPC strategy and add Kafka server
+
+[source, shell]
+----
+cat <<EOF >${OPENNMS_HOME}/etc/opennms.properties.d/kafka.properties
+org.opennms.core.ipc.rpc.strategy=kafka
+org.opennms.core.ipc.rpc.kafka.bootstrap.servers=kafka-1:9092,kafka-2:9092,kafka-3:9092
```

Comment:
It is "safe" to add more than one, but that's optional. Like Cassandra, the driver will attempt to connect to the first entry, if that is successful, the whole broker topology will be discovered and will be well known by the client. The other entries are used only if the connection against the first fails.

Reply:
I have added at the beginning a Note which explains what happens if you add more than one Kafka server to the configuration. So we don't have to explain it twice for RPC and Sink. See here https://github.com/OpenNMS/opennms/pull/2797/files#diff-bf076a743d80c44eec41efc3d95ce27fR31

Reply:
For the reason they are not required to get you going with a Kafka setup and to avoid to clutter the step by step instructions with a lot of hints here, notes there, I've collected them in this section here. The focus in this section is the RPC and Sink feature with Kafka. I would say, the integration with Kafka for integration to have OpenNMS a producer for Alarms, Metrics and Inventory data should be a different topic in my opinion. What do you think=